ci/wheel_smoke_test_distributed_ucxx.py

```diff
@@ -0,0 +1,65 @@
+import asyncio
+
+import pytest
+
+from distributed.comm import connect, listen
+from distributed.protocol import to_serialize
+
+import ucxx
+
+import distributed_ucxx  # noqa: E402
+from distributed_ucxx.utils_test import gen_test, ucxx_loop
+
+
+try:
+    HOST = ucxx.get_address()
+except Exception:
+    HOST = "127.0.0.1"
+
+
+async def get_comm_pair(
+    listen_addr=f"ucxx://{HOST}", listen_args=None, connect_args=None, **kwargs
+):
+    listen_args = listen_args or {}
+    connect_args = connect_args or {}
+    q = asyncio.queues.Queue()
+
+    async def handle_comm(comm):
+        await q.put(comm)
+
+    listener = listen(listen_addr, handle_comm, **listen_args, **kwargs)
+    async with listener:
+        comm = await connect(listener.contact_address, **connect_args, **kwargs)
+        serv_comm = await q.get()
+        return (comm, serv_comm)
+
+
+@pytest.mark.parametrize(
+    "g",
+    [
+        lambda cudf: cudf.Series([1, 2, 3]),
+        lambda cudf: cudf.DataFrame({"a": [1, 2, None], "b": [1.0, 2.0, None]}),
+    ],
+)
+@gen_test()
+async def test_ping_pong_cudf(ucxx_loop, g):
+    # if this test appears after cupy an import error arises
```

Comment:
I assume typo?
```suggestion
    # if this test appears after cudf an import error arises
```

Reply:
Hmm but this is also potentially a bit concerning. I see Python 3.7 in your traceback. Have we tested this on a few different machines with different Python versions? Could we remove this check and try running in CI? I'm wondering if this problem arises due to compatibility issues with the CXX ABI. It's probably not actionable, but it's worth double-checking in case it is.

Reply:
This is just a copy of https://github.com/rapidsai/ucxx/blob/e492dca5b3fe1b0da095c12327b1e1f9c92f262d/python/distributed-ucxx/distributed_ucxx/tests/test_ucxx.py#L178-L215 with fewer parameters, the comment is not relevant here as this test is running alone here so I've removed it inhttps://github.com/rapidsai/ucxx/pull/167/commits/2107f160a6c7530e39dfdcc7d03b5bb124791f5c .